client/src/sections/quests/Level7.tsx

```diff
@@ -24,21 +24,21 @@ const getZoneDisplay = (
 ): JSX.Element | undefined => {
   if (evil > 0) {
     return (
-      <Line href="/crypt.php">
+      <Box key={zone} href="/crypt.php">
```

Comment:
no key necessary here if you make the `mainElement` part below not use a list. also, keys should be added where the list is created, not in an intermediate function